website/content/docs/api-gateway.mdx

```diff
@@ -0,0 +1,171 @@
+---
+layout: docs
+page_title: API Gateway
+description: Using Consul API gateway functionality
+---
+
+# Consul API Gateway
+
+This topic describes how to use the Consul API Gateway module, which enables applications and services running in a datacenter to be accessed by network clients outside the datacenter. This type of network traffic is commonly referred to as "north-south" network traffic.
+
+## Introduction
+
+Consul API Gateway is an implementation of the Kubernetes Gateway [API Specification](https://gateway-api.sigs.k8s.io/). The specification defines a set of custom resource definitions (CRD) that can create logical gateways and routes.
+
+## Requirements
+
+Your datacenter must meet the following requirements prior to configuring the Consul API Gateway:
+
+- A Kubernetes cluster must be running
+- Consul 1.11.1+
+
+## Installation
+
+1. Create a values file for your Consul server agents that contains the following parameters:
+
+   ```yaml
```

Comment:
Adding the codeblocks config to specify that the filename we are using for this is `values.yaml`

```suggestion
<CodeBlockConfig hideClipboard filename="values.yaml">

   ```yaml
```